docs/guides/search.rst

```diff
@@ -29,21 +29,21 @@ In addition to all parameters accepted by ``elgg_get_entities()``, ``elgg_search
 .. code:: php
 
 	// List all users who list United States as their address or mention it in their description
-	$options = [
-		'type' => 'user',
-		'query' => 'us',
-		'fields' => [
-		   'metadata' => ['description'],
-           'annotations' => ['location'],
+    $options = [
```

Comment:
i believe this should all be tabs... am i right? Maybe a misconfigured rst editor?

Reply:
RST sucks